content/doc/tutorials/_run-jenkins-in-docker.adoc

```diff
@@ -0,0 +1,125 @@
+
+=== Run Jenkins in Docker
+
+In this tutorial, you'll be running Jenkins as a Docker container from the
+link:https://hub.docker.com/r/jenkinsci/blueocean/[`jenkinsci/blueocean`] Docker
+image. Read more about these concepts in the
+link:/doc/book/installing/#docker[Docker] and
+link:/doc/book/installing/#downloading-and-running-jenkins-in-docker[Downloading
+and running Jenkins in Docker] sections of the
+link:/doc/book/installing/[Installing Jenkins] page.
+
+*Tip:* If you've already run though this procedure (perhaps from link:..[another
+introductory tutorial]), then you can skip this section and
+<<fork-and-clone-the-sample-repository-on-github,jump to the next>>. However,
+if it's been a while since you last ran through this procedure, then it is
+recommended that you re-run `docker run ...` command below again (if you haven't
+already done so recently) to acquire any updates to this Docker image.
+
+. Open up a terminal/command prompt window.
+. Run the `jenkinsci/blueocean` image as a container in Docker using the
+  following
+  link:https://docs.docker.com/engine/reference/commandline/run/[`docker run`]
+  command (bearing in mind that this command automatically downloads the image
+  if this hasn't been done):
+** *On macOS and Linux:*
++
+[source]
+----
+docker run \
+  --rm \
+  -u root \
+  -p 8080:8080 \
+  -v jenkins-data:/var/jenkins_home \ # <1>
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  -v "$HOME":/home \ # <2>
+  jenkinsci/blueocean
+----
+<1> Maps the `/var/jenkins_home` directory in the container to the Docker
+link:https://docs.docker.com/engine/admin/volumes/volumes/[volume] with the name
+`jenkins-data`. If this volume does not exist, then this `docker run` command
+will automatically create the volume for you.
+<2> Maps the `$HOME` directory on the host (i.e. your local) machine (usually
+the `/Users/<your-username>` directory) to the `/home` directory in the
+container.
++
+*Note:* If copying and pasting the command snippet above doesn't work, try
+copying and pasting this annotation-free version here:
++
+[source]
+----
+docker run \
+  --rm \
+  -u root \
+  -p 8080:8080 \
+  -v jenkins-data:/var/jenkins_home \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  -v "$HOME":/home \
+  jenkinsci/blueocean
+----
+
++
+** *On Windows:* +
+  The following command is effectively the same command shown for macOS/Linux
+  above but the `-v "$HOME":/home` is replaced with `-v "%HOMEPATH%":/home`
+  (assuming that your user home directory on Windows is on the C drive). Also,
+  if you're using the Windows command prompt, replace the backslash symbols `\`
+  at the end of each line with carat symbols instead `^`, giving you:
++
+----
+docker run ^
+  --rm ^
+  -u root ^
+  -p 8080:8080 ^
+  -v jenkins-data:/var/jenkins_home ^
+  -v /var/run/docker.sock:/var/run/docker.sock ^
+  -v "%HOMEPATH%":/home ^
+  jenkinsci/blueocean
+----
+
++
+If you have some experience with Docker, you can add an option like `--name
```

Comment:
Same as above. Give reason.

Reply:
Done - like I did for the main installation instructions.